classes/checkout/CheckoutPaymentStep.php

```diff
@@ -35,16 +35,22 @@ public function handleRequest(array $requestParams = array())
 
     public function render(array $extraParams = array())
     {
-        return $this->renderTemplate(
-            $this->getTemplate(), $extraParams, array(
-                'payment_options' => $this
-                    ->paymentOptionsFinder
-                    ->getPaymentOptionsForTemplate(),
-                'conditions_to_approve' => $this
-                    ->conditionsToApproveFinder
-                    ->getConditionsToApproveForTemplate(),
-                'selected_payment_option' => $this->selected_payment_option,
-            )
-        );
+        $delivery_options = $this->getCheckoutSession()->getDeliveryOptions();
```

Comment:
When I said `camelCase 😢 ?` I meant: can you put them all in camelCase? No more snake_case in PHP 😄